src/lib.rs

```diff
@@ -94,7 +94,7 @@
 //! declare it up front. But if you don't, things will still work.
 
 #![deny(missing_debug_implementations, nonstandard_style)]
-#![warn(missing_docs, unreachable_pub)]
```

Comment:
Why this change? It should be `pub(crate)` instead then, no?

Reply:
The lint triggers lots of false positives. We've gone through enabling / disabling it in the past. For whatever reason it was flagging `cache::ClearSiteData` as "private" when it's clearly public, so I think it's time to disable the lint again.